### PR TITLE
Stripped binaries, updated Grafana, and updated Istio codebase to 1.0.1 

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: istio-proxy/istio-proxy.spec [hunk 1]
```diff
@@ -21,7 +23,7 @@
 
 Name:           istio-proxy
 Version:        0.1.0
-Release:        1%{?dist}
+Release:        2%{?dist}
```
Comment (dmitri-d) [inline istio-proxy/istio-proxy.spec]: Should this be version 0.1.1 and release 1 instead?

Comment (brian-avery) [reply istio-proxy/istio-proxy.spec]: Good question. I'm not sure.

File: istio-proxy/istio-proxy.spec [hunk 2]
```diff
@@ -2,14 +2,16 @@
 %global with_devel 0
 # Build with debug info rpm
 %global with_debug 0
+#Build stripped binaries
+%global with_stripped_binaries 1
```
Comment (dmitri-d) [inline istio-proxy/istio-proxy.spec]: Couldn't we use with_debug? I think with_debug == 0 implies stripping of binaries. Alternatively, we could move all devel/debug stuff into a dedicated package...

File: istio-proxy/istio-proxy.spec (hunk 1 above)
Comment (brian-avery) [reply istio-proxy/istio-proxy.spec]: It's a new Istio release that we're picking up. It just happens to be bug fixes.

**Pull request closed without merging**